models/time_cache.cr

```diff
@@ -0,0 +1,26 @@
+struct TimeCache(K, V)
```

Comment:
I like this, but I'm wondering if it belongs here or in Frank. What do you think?